adafruit_pcf8591/analog_out.py

```diff
@@ -26,12 +26,16 @@
 
 * Author(s): Bryan Siepert
 """
+try:
+    import typing  # pylint: disable=unused-import
+except ImportError:
+    pass
 
 
 class AnalogOut:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, dac_pin=0):
+    def __init__(self, pcf: "PCF8591", dac_pin: int = 0) -> None:
```

Comment:
You should still be able to reference `adafruit_pcf8591.pcf8591.PCF8591` here as well.